modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -24,20 +24,20 @@ An Operator with the condition type `Available` is functional and available in t
 +
-Operators do not report the condition type `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the Operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
+Operators do not report the condition type `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the Operator is reacting to it, then the status reports back as `True`, since it is moving from one steady state to another.
 +
 * Degraded:
-An Operator with the condition type `Degraded` has a current state that does not match the required state over a period of time. The period of time can vary by component, but a `Degraded` state represents persistent observation of an Operator's condition.  As a result, an Operator will not fluctuate in and out of the `Degraded` state.  
+An Operator with the condition type `Degraded` has a current state that does not match the required state over a period of time. The period of time can vary by component, but a `Degraded` state represents persistent observation of an Operator's condition.  As a result, an Operator does not fluctuate in and out of the `Degraded` state.  
 +
 There might be a different condition type if the transition from one state to another does not persist over a long enough period to report `Degraded`.  
-An Operator will not report `Degraded` during the course of a normal upgrade.  An Operator may report `Degraded` in response to a persistent infrastructure failure that requires eventual administrator intervention.
+An Operator does not report `Degraded` during the course of a normal upgrade.  An Operator may report `Degraded` in response to a persistent infrastructure failure that requires eventual administrator intervention.
 +
 [NOTE]
 ====
 This condition type is only an indication that something may need investigation and adjustment. As long as the Operator is available, the `Degraded` condition does not cause user workload failure or application downtime.
 ====
 +
 * Upgradeable:
-An Operator with the condition type `Upgradeable` indicates whether the Operator is safe to upgrade based on the current cluster state. The message field will contain a human-readable description of what the administrator needs to do for the cluster to successfully update. The CVO allows updates when this condition is `True`, `Unknown` or missing. 
+An Operator with the condition type `Upgradeable` indicates if the Operator is safe for you to perform a minor update based on the current cluster state. The message field contains a human-readable description of what the administrator needs to do for the cluster to successfully update. The CVO allows updates when this condition is `True`, `Unknown` or missing. 
```

Comment:
isn't Uprgadeable=False blocking all types of upgrade in case there's cv overrides set?

Reply:
IIRC, only minor updates are impacted. https://github.com/openshift/openshift-docs/pull/56042#issuecomment-1433786590 @wking can you confirm?

Line 43 should address this:

"When the `Upgradeable` status is `False`, only minor updates are impacted, and the CVO prevents the cluster from performing impacted updates unless forced."

Reply:
ClusterOperator `Upgradeable=False` is only about minor updates.  ClusterVersion `Upgradeable=False` is mostly about minor updates, but can be about patch updates if there are ClusterVersion `overrides` (since [rhbz#1822844][1]).  We get at this [in godocs with][2]:

> When set on ClusterVersion, the message will explain which updates (minor or patch) are impacted. When set on ClusterOperator, False will block minor OpenShift updates.

I'd personally rather leave `overrides` unmentioned in openshift-docs, because you can do dangerous things with it. 
 And I'm agnostic about whether the `Upgradeable` docs here get into "ClusterVersion `Upgradeable=False` could block patches" or leave that as an undocumented behavior for an undocumented `overrides` situation.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1822844
[2]: https://github.com/openshift/api/blob/cce310ad2932f6de24491052d506926e484c082c/config/v1/types_cluster_operator.go#L182-L185

Reply:
> or leave that as an undocumented behavior for an undocumented overrides situation.

actually overrides are mentioned in docs [1]. that page itself however states that overrides block upgrades, so may be enough for such a rare corner case. so, I'm fine with either..
@shellyyang1989 WDYT?


[1]  https://docs.openshift.com/container-platform/4.12/architecture/architecture-installation.html#unmanaged-operators_architecture-installation

Reply:
Thanks Evgeni for raising it to discuss. I personally prefer not to mention overrides in official doc. It's not something we recommend to production clusters.

Reply:
Thanks, all! :) I will keep it as is for now.